frontend/src/pages/HomePage/index.js

```diff
@@ -5,19 +5,23 @@ import Title from "components/Typography/Title";
 
 import ellipse1 from "assets/imgs/ellipses/Ellipse1.svg";
 import styles from "./index.module.css";
+import * as constants from "constants/Subvisual.js";
```

Comment:
You don't have to import everything from the constants file, if all you need is, for example, the `SUBVISUAL` constant.

Also, consider object destructuring here too.